mapadroid/utils/walkerArgs.py

```diff
@@ -370,6 +370,23 @@ def parse_args():
     parser.add_argument('-rrqi', '--redis_report_queue_interval', default=30, type=int,
                         help='Report queue size from mitmreciver to redis every every N seconds (Default: 30)')
 
+    # PTC Login tracking
+    parser.add_argument('-elt', '--enable_login_tracking', action='store_true', default=False,
+                        help=('Enable tracking of login attempts to PTC'))
+    parser.add_argument('-lth', '--login_tracking_host', default='localhost',
```

Comment:
Is this another redis instance rather than using buckets or unique keys not used in another part of MAD?

Reply:
Yes, because this is a 1:1 port of a version of this I had made for master/legacy some time ago, where redis was not mandatory.

Reply:
Nonetheless. I use a single redis for tracking in multiple mad instances, so it would still need to be configurable